services/searchengine/bleveengine/bleve_test.go

```diff
@@ -113,7 +114,7 @@ func (s *BleveEngineTestSuite) TestDeleteChannelPosts() {
 		channelToAvoidID := model.NewId()
 		posts := make([]*model.Post, 0)
 		for i := 0; i < 10; i++ {
-			post := createPost(userID, channelID, "test one two three")
+			post := createPost(userID, channelID, "test one two three "+time.Now().String())
```

Comment:
unrelated?

Reply:
In every case `"test one two three"`

Reply:
Let's use `model.NewRandomString` for this.